README.md

```diff
@@ -13,8 +13,9 @@ An LSP server provides rich information about source code. An LSP client consume
 ```swift
 import SwiftLSPClient
 
-let executablePath = "path/to/your/executable-lsp-server"
-let host = LanguageServerProcessHost(path: executablePath, arguments: [])
+let executablePath = "path/to/your/lsp-server-executable"
+let host = LanguageServerProcessHost(path: executablePath, arguments: [],
+    environemnt: [/* the environment your lsp server requires e.g. PATH */])
```

Comment:
Yeah, very good idea. Sorry again this was so hard to figure out.

Reply:
👍

Reply:
I don't immediately see how to add references to TextDocumentClientCapabilities. Isn't it a workspace operation?